dev/components/menu/tpl.pug

```diff
@@ -1,26 +1,26 @@
 .app-menu(ng-class="vm.className")
   .menu-button(
     ng-click="vm.redirect('/dashboard')"
-    ng-class="{active: vm.url === '/dashboard'}")
+    ng-class="{active: vm.url.includes('/dashboard')}")
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/includes
Problems with compatibility

Reply:
Should I apply polyfill?